src/js/components/RangeSelector/RangeSelector.js

```diff
@@ -375,13 +375,18 @@ const RangeSelector = forwardRef(
           <Text
             ref={minRef}
             textAlign="end"
-            size="small"
-            margin={{ horizontal: 'small' }}
+            // do we need theme for size?
```

Comment:
Prop names look good, but I have the same question. Do we need to have a theme prop for size?

Reply:
Good question -- I don't think text size is required in the scope of "breaking t-shirt sizes" (since we're more worried about margin/pad/gap.

I'd be fine if we want to leave as just the margin one for now, unless you think adding text is useful for flexibility.